app/src/main/java/com/duckduckgo/app/browser/favicon/FaviconManager.kt

```diff
@@ -64,10 +64,14 @@ class DuckDuckGoFaviconManager constructor(
     }
 
     override suspend fun loadToViewFromPersisted(url: String, view: ImageView) {
+        // avoid displaying the previous favicon when the holder is recycled
+        faviconDownloader.loadDefaultFaviconToView(view)
```

Comment:
I think we can move these calls inside `loadFaviconToView` in the `GlideFaviconDownloader`

Reply:
Yeah, I thought about that. Left it like this to avoid having `loadToViewFromDirectory` doing more than just loading the icon from the directory, it could an undesired side effect to reset the icon in the view when calling `loadToViewFromDirectory` if we call it from other places in the future.
What do you think?
Anyway given that that's not currently the case and that it is a private method I am also ok with moving it into `loadToViewFromDirectory` if you feel strongly about it.

Reply:
Nah, not feeling strongly about it :)